js/video-controller.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Class representing the main menu navigation.
+ */
+export default class VideoController {
+  constructor(classEl) {
+    let videoWrapper = document.querySelector(
+      "." + classEl + "__controller-wrapper"
```

Comment:
This might be a bit too specific and box us in from reuse standpoint. Why don't we just treat `classEl` as a whole selector, and change line 8 in index.js to be `new VideoController('.hero-video__controller-wrapper')`. That way it could also take a data attribute selector, ID, or anything else.